lib/engine/part/path.rb

```diff
@@ -36,18 +37,20 @@ def self.make_lanes(a, b, terminal: nil, lanes: nil, a_lane: nil, b_lane: nil, t
                      terminal: terminal,
                      lanes: [a_lanes, b_lanes],
                      track: track,
+                     gauge: gauge || track,
```

Comment:
not necessary since you do that on line 63,  do you even need a property? it could just be a method gauge

Reply:
Removed gauge completely from this PR.